library/lua/gui/widgets.lua

```diff
@@ -30,9 +30,38 @@ DimensionsTooltip = require('gui.widgets.dimensions_tooltip')
 Tab = TabBar.Tab
 makeButtonLabelText = Label.makeButtonLabelText
 
-DOUBLE_CLICK_MS = Panel.DOUBLE_CLICK_MS
-STANDARDSCROLL = Scrollbar.STANDARDSCROLL
-SCROLL_INITIAL_DELAY_MS = Scrollbar.SCROLL_INITIAL_DELAY_MS
-SCROLL_DELAY_MS = Scrollbar.SCROLL_DELAY_MS
+---@return boolean
+function getDoubleClickMs()
+    return Panel.DOUBLE_CLICK_MS
+end
+function setDoubleClickMs(value)
+    Panel.DOUBLE_CLICK_MS = value
+end
+
+---@return boolean
+function getStandardScroll()
+    return Scrollbar.STANDARDSCROLL
+end
+function setStandardScroll(value)
+    Scrollbar.STANDARDSCROLL = value
+end
```

Comment:
STANDARDSCROLL is a constant and doesn't need a function wrapper. we *could* keep `getStandardScroll()` if you think it makes the API clearer, but there should not be a `setStandardScroll()`